src/validation.h

```diff
@@ -107,7 +107,12 @@ enum class SynchronizationState {
 };
 
 extern RecursiveMutex cs_main;
-typedef std::unordered_map<uint256, CBlockIndex*, BlockHasher> BlockMap;
+
+// If we ever switch to another associative container, we need to either use a
```

Comment:
In commit "No more heap BlockIndices" (6a897c0601dff4f116b1ab223595c5c2fac0e7a3)

Comment seems to lack a little context, maybe prefix with "Because validation code takes pointers to the map's CBlockIndex objects, if we ever switch..."